Fails to build without the snap feature enabled

`cargo auditable install crabz --no-default-features --features=deflate_rust` results in the following error:

```
error[E0599]: no variant or associated item named `Snap` found for enum `Format` in the current scope
   --> /home/shnatsel/.cargo/registry/src/github.com-1ecc6299db9ec823/crabz-0.8.1/src/main.rs:264:21
    |
150 | enum Format {
    | ----------- variant or associated item `Snap` not found for this enum
...
264 |             Format::Snap => ("sz", string_set!["sz", "snappy"]),
    |                     ^^^^ variant or associated item not found in `Format`

For more information about this error, try `rustc --explain E0599`.
```

--- a/src/main.rs
+++ b/src/main.rs
@@ -261,6 +261,7 @@ impl Format {
             #[cfg(feature = "any_zlib")]
             Format::Zlib => ("zz", string_set!["zz", "z", "gz"]),
             Format::RawDeflate => ("deflate", string_set!["deflate", "zz", "z"]),
+            #[cfg(feature = "snappy")]
             Format::Snap => ("sz", string_set!["sz", "snappy"]),
         }
     }

